Backend/Controllers/ProductsController.cs

```diff
@@ -74,7 +74,7 @@ public async Task<ActionResult<ProductDTO>> GetById(int id)
         }
 
         [HttpPost]
-        public async Task<ActionResult<ProductDTO>> PostProduct([FromBody] ProductDTO product)
```

Comment:
I saw that some project had different DTOs for different API calls https://github.com/digitopoi/DatingApp.API/tree/master/Dtos. We can discuss why they do this and if we should do that also during Monday.